tools/ci_build/github/azure-pipelines/templates/android-java-api-aar.yml

```diff
@@ -55,6 +55,13 @@ jobs:
     clean: true
     submodules: none
 
+  - task: JavaToolInstaller@0
+    displayName: Use jdk 17
+    inputs:
+      versionSpec: '17'
+      jdkArchitectureOption: 'x64'
+      jdkSourceOption: 'PreInstalled'
+
```

Comment:
Does this change the Java version targeted or just the tools. i.e. can you use Java 11 still as Java 17 is only Android 14 (API 34) and later.

https://developer.android.com/build/jdks#compileSdk

Reply:
Actually, the JDK I meant to update is within the docker image, not the host machine. So, this is not needed.